vkcs/internal/services/compute/v2/flavors/results.go

```diff
@@ -0,0 +1,52 @@
+package flavors
+
+import (
+	"encoding/json"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/flavors"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+type FlavorMissingFields struct {
+	ExtraSpecs map[string]interface{} `json:"extra_specs"`
+}
+
+// FlavorWithExtraSpecs needs for extract ExtraSpecs from flavors.FlavorPage
```

Comment:
do not specify extraspecs here - it would be wrong with another "missing" fields in the struct above

Reply:
done